app/services/manifest_builder/services_builder.rb

```diff
@@ -0,0 +1,22 @@
+class ManifestBuilder
+  class ServicesBuilder
+    attr_reader :record
+
+    def initialize(record)
+      @record = record
+    end
+
+    def apply(manifest)
+      return if
+        record.class == AllCollectionsPresenter ||
+        record.class == CollectionShowPresenter
+      service_array = {
+        "@context"  => "http://iiif.io/api/search/0/context.json",
+        "@id"       => "http://localhost:3000/search/#{record.id}",
```

Comment:
I'm concerned about the hard-coded host and port.

Reply:
Ooops. Thought I got that out of there. Was using it for testing. Will correct. Thx.